src/modules/Dashboard/api/api.ts

```diff
@@ -154,6 +156,34 @@ export const revokeAppletUserApi = (
     signal,
   });
 
+export const removeManagerAccess = (
```

Comment:
Could you pls also delete unnecessary old api functions and types regarding access if they exist?